src/components/buttons/ButtonSecondary.jsx

```diff
@@ -19,7 +19,7 @@ export const BUTTON_SECONDARY_TYPE = {
   ALT_INVERSE: 'alt-inverse',
   DARK_INVERSE: 'dark-inverse',
   ACTIVE_INVERSE: 'active-inverse',
-  PEACH: 'peach'
+  ACTIVE_MUSTRAD: 'active-mustard'
```

Comment:
I add this type to have a separate styling for `MARK AS BEST ANSWER` button.

Reply:
you have a typo here - mustRAd